Free products are not checked for accessability anymore (#989)
uyuni-project/uyuni-docs: modules/installation/pages/setup-wizard.adoc

Filters help you search for products by description or architecture.

The list is organized by product name showing products on top which have a subscription.

> You will always see products which are freely available at the end of the list.

Reviewer: ```suggestion
Freely available products appear at the end of the list.
```

Depending on the products you have chosen, synchronization can take up to several hours.
New products will be available for you to use in {productname} when synchronization is complete.

> If your synchronization fails, it could be because of a third party GPG key or your company firewall block access to the download server.

Reviewer 2: ```suggestion
If your synchronization fails, it could be because of a third party GPG key or your company firewall blocking access to the download server.
```
